File: C#/BluffinMuffin.Protocol.DataTypes/Enums/LimitTypeEnum.cs
using System.ComponentModel;

namespace BluffinMuffin.Protocol.DataTypes.Enums
{
    /// <summary>
    /// 
    /// </summary>
    public enum LimitTypeEnum
    {
        /// <summary>
        /// NoLimit
        /// </summary>
        [Description("No Limit")]
        NoLimit,

        /// <summary>
        /// FixedLimit
        /// </summary>
        [Description("Fixed Limit")]
        FixedLimit,

        /// <summary>
        /// PotLimit
        /// </summary>
        [Description("Pot Limit")]
        PotLimit
    }
}

File: Python/BluffinMuffin.Protocol/DataTypes/Enums/LimitTypeEnum.py
from enum import Enum

__author__ = 'ericm'
_project_ = 'BluffinMuffin.Protocol'

class LimitTypeEnum(Enum):
    NoLimit = 0
    FixedLimit = 1
    PotLimit = 2

    def parse(str):
        return LimitTypeEnum[str]

    def to_string(value):
        return value.name